lib/commons/color/get-background-color.js

```diff
@@ -231,33 +231,42 @@ color.getCoords = function(rect) {
  */
 color.getRectStack = function(elm) {
 	let boundingCoords = color.getCoords(elm.getBoundingClientRect());
-	if (boundingCoords) {
-		let boundingStack = dom.shadowElementsFromPoint(
-			boundingCoords.x,
-			boundingCoords.y
-		);
-		// allows inline elements spanning multiple lines to be evaluated
-		let rects = Array.from(elm.getClientRects());
-		if (rects && rects.length > 1) {
-			let filteredArr = rects
-				.filter(rect => {
-					// exclude manual line breaks in Chrome/Safari
-					return rect.width && rect.width > 0;
-				})
-				.map(rect => {
-					let coords = color.getCoords(rect);
-					if (coords) {
-						return dom.shadowElementsFromPoint(coords.x, coords.y);
-					}
-				});
-			// add bounding client rect stack for comparison later
-			filteredArr.splice(0, 0, boundingStack);
-			return filteredArr;
-		} else {
-			return [boundingStack];
-		}
+	if (!boundingCoords) {
+		return null;
 	}
-	return null;
+
+	let boundingStack = dom.shadowElementsFromPoint(
+		boundingCoords.x,
+		boundingCoords.y
+	);
+
+	let rects = Array.from(elm.getClientRects());
+	// If the element does not have multiple rects, like for display:block, return a single stack
+	if (!rects || rects.length <= 1) {
+		return [boundingStack];
+	}
+
+	// Handle inline elements spanning multiple lines to be evaluated
+	let filteredArr = rects
+		.filter(rect => {
+			// exclude manual line breaks in Chrome/Safari
+			return rect.width && rect.width > 0;
```

Comment:
nit: Why not just `return rect.width > 0`?

Reply:
We could. I just changed indentation on this. Didn't actually look to refactor.